src/components/themed/WalletSyncCircle.js

```diff
@@ -24,36 +27,35 @@ export const WalletSyncCircle = (props: Props) => {
   const theme = useTheme()
   const { size = theme.rem(2), edgeWallet } = props
   // Animation shared state
-  const syncRatio = useSharedValue(edgeWallet.syncRatio < 0.05 ? 0.05 : edgeWallet.syncRatio)
-  const isDone = useSharedValue(false)
+  const syncRatio = useSharedValue(edgeWallet.syncRatio < BASE_RATIO ? MIN_RATIO : edgeWallet.syncRatio)
+  const isDone = useRef(false)
   const stroke = useSharedValue(theme.walletProgressIconFill)
 
   // Subscribe to the sync ratio:
   useEffect(
     () =>
       edgeWallet.watch('syncRatio', (ratio: number) => {
         // If already done but needs to resync reset the flags and animations
-        if (isDone.value && ratio < BASE_RATIO) {
-          isDone.value = false
+        if (isDone.current && ratio < BASE_RATIO) {
+          isDone.current = false
           stroke.value = theme.walletProgressIconFill
           syncRatio.value = BASE_RATIO
         }
         // If the wallet hasn't fully synced show progress animation
-        if (!isDone.value) {
-          if (ratio === 1) {
-            isDone.value = true
-            stroke.value = theme.primaryText
+        if (!isDone.current) {
+          if (ratio > DONE_RATIO) {
+            isDone.current = true
+            stroke.value = theme.walletProgressIconDone
           }
           if (ratio < BASE_RATIO) syncRatio.value = withTiming(BASE_RATIO, { duration: 1000 })
-          else if (ratio > MAX_RATIO && ratio < 1) syncRatio.value = withTiming(MAX_RATIO, { duration: 1000 })
+          else if (ratio > MAX_RATIO && ratio < DONE_RATIO) syncRatio.value = withTiming(MAX_RATIO, { duration: 1000 })
```

Comment:
Please just let `syncRatio.value` be the true sync ratio (with animation). The visual minimum & maximum should be implemented using in your `useAnimatedProps` hook using `Math.max(syncRatio.value, MIN_RATIO)` or such. This separates the business logic (tracking the core state) from the display logic. I don't think the max ratio is needed - it was an artifact of the way the old circle component worked.